src/common/index.cc

```diff
@@ -68,13 +68,14 @@ Index<T>::Add(const DataSet& dataset, const Json& json) {
 
 template <typename T>
 inline expected<DataSetPtr>
-Index<T>::Search(const DataSet& dataset, const Json& json, const BitsetView& bitset) const {
+Index<T>::Search(const DataSet& dataset, const Json& json, const BitsetView& _bitset) const {
```

Comment:
Try not to define variables beginning with underscores. These may be reserved by OS and others.

Reply:
fixed.